src/Sensors/ExceptionSensor.php

```diff
@@ -102,9 +103,22 @@ private function wasManuallyReported(Throwable $e): bool
      */
     private function serializeTrace(Throwable $e): string
     {
-        $trace = [];
+        $trace = [
+            // Insert the exception location as the first frame.
+            // This matches the behavior of Symfony's exception renderer.
+            [
+                'file' => $this->location->normalizeFile($e->getFile()).':'.$e->getLine(),
+                'source' => '',
+            ],
+        ];
```

Comment:
This is similar to Symfony's approach that we use in Laravel:

https://github.com/symfony/error-handler/blob/0b31a944fcd8759ae294da4d2808cbc53aebd0c3/Exception/FlattenException.php#L304-L339.